src/data.js

```diff
@@ -2064,6 +2064,27 @@ const pages = [
       'YouTuber',
     ],
   },
+  {
+    name: 'Muhammad Umair',
+    description:
+      'Fullstack front-end developer and designer, passionate in everything #JavaScript',
+    url: 'https://www.linkedin.com/in/mumairofficial',
+    twitter: '@imumare',
+    emoji: '💻',
+    country: 'PK',
```

Comment:
I see PK is there in supported coutries list;

![image](https://user-images.githubusercontent.com/6498132/72229990-6dd16600-35d4-11ea-83cf-220204290710.png)

Reply:
it's also available in https://github.com/wesbos/awesome-uses/blob/93b8ad6856184c4b80406b1c3c09b5c4bc542167/scripts/flags.js#L163

Reply:
It's actually unicode emoji code `🇵🇰` not `PK`. try to copy-paste from `flags.js` once.